add deprecation warning to ubuntu 16.04 docs (#3801)
mattermost/docs: source/install/install-ubuntu-1604-server.rst

Installing Ubuntu Server 16.04 LTS
==================================

.. warning::
  Ubuntu 16.04 is approaching its End of Life in April 2021.

>   We don't recommend installing new instances of Mattermost on Ubuntu 16.04 and instead advise you to go with a newer release like Ubuntu 20.04.

Reviewer: ```suggestion
  Because of this, we don't recommend installing new instances of Mattermost on Ubuntu 16.04. Rather use a newer version such as Ubuntu 20.04.
```
